pkg/abstractions/image/build.go

```diff
@@ -504,6 +463,50 @@ func ExtractImageNameAndTag(imageRef string) (BaseImage, error) {
 	}, nil
 }
 
+func getPythonInstallCommand(pythonVersion string) string {
+	baseCmd := "apt-get update -q && apt-get install -q -y software-properties-common gcc curl git"
+	components := []string{
+		"python3-future",
+		pythonVersion,
+		fmt.Sprintf("%s-distutils", pythonVersion),
+		fmt.Sprintf("%s-dev", pythonVersion),
+	}
+
+	installCmd := strings.Join(components, " ")
+	installPipCmd := fmt.Sprintf("curl -sS https://bootstrap.pypa.io/get-pip.py | %s", pythonVersion)
+	postInstallCmd := fmt.Sprintf("rm -f /usr/bin/python && rm -f /usr/bin/python3 && ln -s /usr/bin/%s /usr/bin/python && ln -s /usr/bin/%s /usr/bin/python3 && %s", pythonVersion, pythonVersion, installPipCmd)
+
+	return fmt.Sprintf("%s && add-apt-repository ppa:deadsnakes/ppa && apt-get update && apt-get install -q -y %s && %s", baseCmd, installCmd, postInstallCmd)
+}
+
+func generatePipInstallCommand(pythonPackages []string, pythonVersion string) string {
+	flagLines, packages := parseFlagLinesAndPackages(pythonPackages)
+
+	command := fmt.Sprintf("%s -m pip install --root-user-action=ignore", pythonVersion)
+	if len(flagLines) > 0 {
+		command += " " + strings.Join(flagLines, " ")
+	}
+	if len(packages) > 0 {
+		command += " " + strings.Join(packages, " ")
+	}
+
+	return command
+}
+
+func generateMicromambaInstallCommand(pythonPackages []string) string {
+	flagLines, packages := parseFlagLinesAndPackages(pythonPackages)
+
+	command := fmt.Sprintf("%s install -y -n beta9", micromambaCommandType)
+	if len(flagLines) > 0 {
+		command += " " + strings.Join(flagLines, " ")
+	}
+	if len(packages) > 0 {
+		command += " " + strings.Join(packages, " ")
+	}
+
+	return command
+}
+
```

Comment:
None of this changed. I moved it because it was a method on the BuildService, but didn't need to be.